ci-operator/step-registry/additional-enabled-capabilities/additional-enabled-capabilities-commands.sh

```diff
@@ -0,0 +1,39 @@
+#!/bin/bash
+
+set -o nounset
+set -o errexit
+set -o pipefail
+
+# Use yq to create cluster monitoring config, as other steps may adjust it
+YQ_URI=https://github.com/mikefarah/yq/releases/download/3.3.0/yq_linux_amd64
+YQ_HASH=e70e482e7ddb9cf83b52f5e83b694a19e3aaf36acf6b82512cbe66e41d569201
+echo "${YQ_HASH} -" > /tmp/sum.txt
+if ! curl -Ls "${YQ_URI}" | tee /tmp/yq | sha256sum -c /tmp/sum.txt >/dev/null 2>/dev/null; then
+  echo "Expected file at ${YQ_URI} to have checksum ${YQ_HASH} but instead got $(sha256sum </tmp/yq | cut -d' ' -f1)"
+  strings /tmp/yq
+  exit 1
+fi
+echo "Downloaded yq; sha256 checksum matches expected ${YQ_HASH}."
+chmod +x /tmp/yq
+
+CONFIG="${SHARED_DIR}/install-config.yaml"
+
+if [ -z "${ADDITIONAL_CAPABILITY_SET}" ]; then
+    echo "Not set envrionment variable ADDITIONAL_CAPABILITY_SET, exiting"
+    exit 1
+fi
+
+if [ -z "${BASELINE_CAPABILITY_SET}" ]; then
+    echo "additionalEnabledCapabilities extends the additional capabililities beyoind what you set in baselineCapabilitySet, please also set envrionment variable BASELINE_CAPABILITY_SET, exiting"
+    exit 1
+fi
+
+#select one optional operator from ${additional_caps_set} for testing
+additional_caps_set=(${ADDITIONAL_CAPABILITY_SET})
+additional_caps=${#additional_caps_set[@]}
+selected_optional_operator_index=$((RANDOM % ${additional_caps}))
+target_optional_operator=${additional_caps_set[$selected_optional_operator_index]}
+echo "$(date -u --rfc-3339=seconds) - Selected optional operator: ${target_optional_operator}"
+
+#add parameter additionalEnabledCapabilities into install-config.yaml file
+/tmp/yq w --style tagged -i "${CONFIG}" 'capabilities.additionalEnabledCapabilities[+]' "${target_optional_operator}"
```

Comment:
I don't see `baselineCapabilitySet` configuration here.

Reply:
`baselineCapabilitySet` setting is here https://github.com/openshift/release/blob/master/ci-operator/step-registry/ipi/conf/ipi-conf-commands.sh#L38, the step of `additional-enabled-capabilities` only focus on configuring `additionalEnalbedCapabilities`.

Reply:
It's in `ipi-conf`, ignore my above comments.